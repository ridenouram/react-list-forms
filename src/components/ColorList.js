import React from 'react';
import Color from './Color';
import PropTypes from 'prop-types';

export default function ColorList({ colors }) {
  ColorList.PropTypes = {
    colors: PropTypes.array.isRequired
  };

  const colorList = colors.map(({ name, hex, rgb }) => {
    
  })

  return (
    <ul>
      
    </ul>
  )
}
