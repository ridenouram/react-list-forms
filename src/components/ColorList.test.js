import React from 'react';
import { shallow } from 'enzyme';
import ColorList from './ColorList';

describe('ColorList component', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<ColorList />);
    expect(wrapper).toMatchSnapshot();
  });
});
