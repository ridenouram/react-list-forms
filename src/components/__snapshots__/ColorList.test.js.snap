// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorList component matches snapshot 1`] = `
<Fragment>
  <CreateColor
    addColor={[Function]}
  />
  <Colors
    colors={Array []}
  />
</Fragment>
`;
